### ISSUE TITLE
Notifications for number of grids on the page should be removed

### ISSUE THREAD

**Issue by helenvholmes** [mozilla/css-grid-inspector#19]
Currently the add-on is using the notification paradigm to express number of grids on the page:

<img width="57" alt="screen shot 2016-08-02 at 11 29 31 am" src="https://cloud.githubusercontent.com/assets/1720093/17334479/669aab32-58a4-11e6-8c37-4a2010ecf01c.png">

This is confusing, as notifications are meant to disrupt flow and be cleared out. Because this is a label, and not a true notification, you can't get rid of them. This causes them to come off as an annoyance.

**Comment by jensimmons** [comment]
I agree. I think we should remove this notification banner. I've honestly found it alarming / disruptive. And it does mix mental models in a way that doesn't work. 

If something is giving me a banner with a notification, that's because it wants me to do something — to take action. Here, there's nothing to do. I can't dismiss it. 

I know for some people this isn't annoying at all. Those people have phones with 3492 unread email banners on their home screen. (AHHHH!!)

Also — this is not something that the new tool will have when we move to Dev Tools proper. So, in the vein of "let's not ship any feature that we won't have later" we shouldn't have this now.

**Comment by potch** [comment]
Would it make sense to instead maybe just have a grayed-out vs non-grayed-out graphic for the button for the presence of grids on the page?

**Comment by jensimmons** [comment]
Oh that's a good idea. Make the button look like it won't do anything when there are no Grids, and it won't do anything. Then make the button look like it does now, like it will work, when it's got something to display. That's much more subtle. I like that better for sure — if there's no performance hit for testing to figure out whether the button should be grey or not. (I don't want our button to cause big performance problems in FF, given how much the company is striving to make FF fast.)
